lib/helper/REST.js

```diff
@@ -0,0 +1,110 @@
+'user strict';
+const Helper = require('../helper');
+const requireg = require('requireg');
+let unirest;
+
+/**
+* REST helper to execute RESTful API calls, this is currently under development phase.
+* Please visit our demo repo to know how to use this Helper.
+* [https://github.com/DigitalOnUs/codeceptjs-poc/tree/master/tests/REST#using-rest-api-on-codeceptjs]
+* Usage:
+*      let response = yield I.sendGet('https://randomuser.me/api/')
```

Comment:
please markdown code formatting tags.
and move this example to `sendGet` method